.gitignore

```diff
@@ -173,4 +173,9 @@ db
 .DS_Store
 
 notebooks/*.yaml
-.ipynb_checkpoints/
\ No newline at end of file
+.ipynb_checkpoints/
+
+# local secret files
+*.sh
+*.pub
+*.crt
```

Comment:
Umm not sure if this is needed since we dont have any of these files in the repository itself. It is probably on your local system and you would want to move them out of the repository directory.

Reply:
Cool